### PR TITLE
Update cam to work with ecflow workflow

### PR DESCRIPTION
**Pull request by lgannoaa**
GitHub #258 

Tested in development ecflow workflow. Result checked by CAM team.

[Yes] Has the code been checked to ensure that no errors occur during the execution? [Yes or No]

### CODE REVIEW

File: scripts/cam/prep/exevs_cam_radar_prep.sh
Comment (MarcelCaron-NOAA) [inline scripts/cam/prep/exevs_cam_radar_prep.sh:1]: Hi @lgannoaa —re: the added MET config variables in this PR.  We recently removed these and similar variables from all scripts that are part of jobs that run METplus in favor of those variables already defined in [EVS/parm/metplus_config/machine.conf](https://github.com/NOAA-EMC/EVS/blob/cb64f5c4967424e642be595bf544059a761b98af/parm/metplus_config/machine.conf).  Is machine.conf not being properly fed to the METplus processes for these radar/severe jobs?

Comment (LinGan-NOAA) [reply scripts/cam/prep/exevs_cam_radar_prep.sh:1]: I am not aware of that change. Please let me know if I should revert this one.

**Comment by ShelleyMelchior-NOAA** [comment]
@lgannoaa , I suggest you make sure your code base for the PR is updated. Whether you use `develop` or `release/evs.v1.0.0`, both versions are up to date and do not have some of the features you are reintroducing w/ this PR, for example what @MarcelCaron-NOAA pointed out.

**Comment by ShelleyMelchior-NOAA** [comment]
@lgannoaa , I think running a sync fork may fix this. I see you are 4 commits behind.

**Comment by lgannoaa** [comment]
@ShelleyMelchior-NOAA I did the sync fork.
@MarcelCaron-NOAA , it the issue resolved?

**Comment by MarcelCaron-NOAA** [comment]
Thanks @lgannoaa.  No the additions are still there.  It doesn't look like the changes ever existed and were removed in a previous PR, only that they are added in this PR.  

Are these changes intentional, based on ecflow needs?

**Comment by lgannoaa** [comment]
@MarcelCaron-NOAA should I remove the following session of the code from all scripts?
export MET_bin_exec=bin
export metplus_verbosity=DEBUG
export met_verbosity=2
export log_met_output_to_metplus=yes
Thanks

**Comment by ShelleyMelchior-NOAA** [comment]
Also, I think this PR should be closed and reopened. All EE2 (or EE2 prep) PRs needs to be for `release/evs.v1.0.0` now, which is the code we are delivering. This PR is into `develop`. We are not delivering what is in `develop`. I think starting w/ a new baseline will fix this. Changes into `release/evs.v1.0.0` will get back into develop.

**Comment by lgannoaa** [comment]
Closed

**Pull request closed without merging**